doc/crypto/OPENSSL_malloc.pod

```diff
@@ -0,0 +1,164 @@
+=pod
+
+=head1 NAME
+
+OPENSSL_malloc_init,
+OPENSSL_malloc, OPENSSL_zalloc, OPENSSL_realloc, OPENSSL_free,
+OPENSSL_clear_realloc, OPENSSL_clear_free,
+CRYPTO_malloc, CRYPTO_zalloc, CRYPTO_realloc, CRYPTO_free,
+OPENSSL_strdup, OPENSSL_strndup, 
+OPENSSL_memdup, OPENSSL_strlcpy, OPENSSL_strlcat,
+CRYPTO_clear_realloc, CRYPTO_clear_free,
+CRYPTO_get_mem_functions, CRYPTO_set_mem_functions,
+CRYPTO_set_mem_debug, CRYPTO_mem_ctrl,
+OPENSSL_mem_debug_push, OPENSSL_mem_debug_pop,
+CRYPTO_mem_debug_push, CRYPTO_mem_debug_pop,
+CRYPTO_mem_leaks - Memory allocation functions
+
+=head1 SYNOPSIS
+
+ #include <openssl/crypto.h>
+
+ #define OPENSSL_malloc_init()
+
+ #define OPENSSL_malloc(num)
+ #define OPENSSL_zalloc(num)
+ #define OPENSSL_realloc(addr, num)
+ #define OPENSSL_free(addr)
+ #define OPENSSL_strdup(str)
+ #define OPENSSL_strndup(str, n)
+ #define OPENSSL_clear_realloc(addr, old_num, num)
+ #define OPENSSL_clear_free(addr, num)
```

Comment:
I think that constants can be presented as `#define` in the SYNOPSIS.  If we have other function macros that are presented as `#define` in the SYNOPSIS, it would seem appropriate to change them, no?